test/common_utils.py

```diff
@@ -280,6 +280,8 @@ def _check_module_exists(name):
 # it felt a little awkward.
 TEST_SKIP_FAST = os.getenv('PYTORCH_TEST_SKIP_FAST', '0') == '1'
 
+UNDER_CUDA_MEMCHECK = os.getenv('UNDER_CUDA_MEMCHECK', '0') == '1'
```

Comment:
Can you put this in common_device_type near the CUDA class? 

Also, can we name it just CUDA_MEMCHECK?

Reply:
Actually, can we name the env var PYTORCH_CUDA_MEMCHECK, to avoid possible collisions?

Reply:
fixed